extensions/data-plane/data-plane-azure-storage/src/test/java/org/eclipse/edc/connector/dataplane/azure/storage/pipeline/AzureStorageDataSinkFactoryTest.java

```diff
@@ -39,8 +42,11 @@
 class AzureStorageDataSinkFactoryTest {
     private final BlobStoreApi blobStoreApi = mock(BlobStoreApi.class);
     private final Vault vault = mock(Vault.class);
+    private final ServiceExtensionContext context = mock(ServiceExtensionContext.class);
```

Comment:
You could use type inference here
```suggestion
    private final ServiceExtensionContext context = mock();
```